### DISCUSSION TITLE
Repository failing to create

### DISCUSSION THREAD

**Discussion by anissacc** [community/community#38624] [Repositories] [answered]
Hi all! I am trying to create a new repository in my private class organization and it is just not loading. It lets me get to the creation page, but the minute I click "create repository" the page takes too long to load and "Unicorn!" pops up. Has anyone else experienced this? I have full permissions to create a repo in my organization and it worked for another student but not me. I've also already checked my internet connections and it fails to load on 2 different networks. Any ideas?

**Comment by ZaptorZap** [comment, marked as answer]
You may wanna review https://www.githubstatus.com/

**Reply by anissacc** [reply]
Omg :(( I have been incessantly checking the status update and now it's only just updated. Oml. Thank you so much lol

**Reply by mdAshik922** [reply]
hmm

**Comment by mdAshik922** [comment]
Server or network problem. Although it appears that the net is doing well. You can try again later

**Reply by anissacc** [reply]
Thank you! I wasn't sure if it was me or the server haha.
